### DISCUSSION TITLE
Mimemagic 0.3.5 bundle locked

### DISCUSSION THREAD

**Discussion by BlackboWeb** [postalserver/postal#1507] [Help with using Postal] [answered]
Hi,
at this step "postal bundle /opt/postal/vendor/bundle", following from the https://github.com/postalserver/postal/wiki/Installation, following error shows up

Your bundle is locked to mimemagic (0.3.5) from rubygems repository
https://rubygems.org/ or installed locally, but that version can no longer be found
in that source. That means the author of mimemagic (0.3.5) has removed it. You'll
need to update your bundle to a version other than mimemagic (0.3.5) that hasn't
been removed in order to install.

I searched for it and got to know that this version is yanked.
Can someone please guide me, how to fix this.

**Comment by willpower232** [comment, marked as answer]
This refers to the old way of installing Postal, you should be able to follow the new documentation to install Postal with The New Way and not encounter this issue.
